ecs-cli/modules/cli/regcreds/regcreds_app.go

```diff
@@ -210,6 +210,9 @@ func validateCredsInput(input regcredio.ECSRegCredsInput, kmsClient kms.Client)
 	if len(inputRegCreds) == 0 {
 		return nil, errors.New("provided credentials must contain at least one registry")
 	}
+	if len(inputRegCreds) > 10 {
```

Comment:
What's the origin of this limit of 10?

Reply:
10 is the max # of containers you can have on a task def, so any more than that and we know there's more than can be used in one app/task/service, which is what we want to scope the role to.

Reply:
perhaps constantize 10 in that case?